SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/TypeStructure.cpp

```diff
@@ -338,11 +338,28 @@ TSharedPtr<FUnrealType> CreateUnrealTypeInfo(UStruct* Type, uint32 ParentChecksu
 		RepDataNode->Condition = Parent.Condition;
 		RepDataNode->RepNotifyCondition = Parent.RepNotifyCondition;
 		RepDataNode->ArrayIndex = PropertyNode->StaticArrayIndex;
+#if ENGINE_MINOR_VERSION >= 25
+		if (Class->IsChildOf(AActor::StaticClass()))
+		{
+			// Uses the same pattern as ComponentReader::ApplySchemaObject and ReceivePropertyHelper
+			if (UNLIKELY((int32)AActor::ENetFields_Private::RemoteRole == Cmd.ParentIndex))
```

Comment:
This is exactly the same. Refactor timeeeeeeeeeeeeeeeeeeeeeeeee?

Reply:
Took a further look, and unfortunately this is not exactly the same, as the actions inside are different.
Could still be refactored with a function taking two lambdas, but I feel like that could be over-complicating it, and Epic also have a couple variations of this code.